### ISSUE TITLE
[1.10.2] Game crash when accessing armor information through gameplay.

### ISSUE THREAD

**Issue by Iteries** [sokratis12GR/ArmorPlus#105]
**ArmorPlus Version**: 1.10.2-8.3.0.1 or more say the 8.4.0.1 but as mentioned in another issue not loading as correct  #one?

**TheDragonLib Version**:  thedragonlib-1.10.2-1.0.3.0

**MinecraftForge Version**: 2221

**Minecraft Version**: 1.10.2

**Client or Server?**: 

- [x] Client

**Crash Report**:  
Time: 2/10/17 8:35 PM
Description: Rendering screen
java.lang.NullPointerException: Rendering screen
	at net.thedragonteam.armorplus.util.PotionUtils.localizePotion(PotionUtils.java:58)
	at net.thedragonteam.armorplus.armors.base.BaseArmor.func_77624_a(BaseArmor.java:401)
	at net.minecraft.item.ItemStack.func_82840_a(ItemStack.java:614)
	at net.minecraft.client.gui.GuiScreen.func_146285_a(GuiScreen.java:136)
	at net.minecraft.client.gui.inventory.GuiContainer.func_73863_a(GuiContainer.java:166)
	at com.rikmuld.corerm.tabbed.GuiTabbed.func_73863_a(GuiTabbed.scala:40)
	at com.rikmuld.camping.inventory.gui.GuiCampinginv.func_73863_a(GuiCamping.scala:131)
	at net.minecraftforge.client.ForgeHooksClient.drawScreen(ForgeHooksClient.java:382)
	at sun.reflect.GeneratedMethodAccessor20.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:483)
	at Reflector.callVoid(Reflector.java:332)
	at net.minecraft.client.renderer.EntityRenderer.func_181560_a(EntityRenderer.java:1400)
	at net.minecraft.client.Minecraft.func_71411_J(Minecraft.java:1076)
	at net.minecraft.client.Minecraft.func_99999_d(Minecraft.java:371)
	at net.minecraft.client.main.Main.main(SourceFile:124)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:483)
	at net.minecraft.launchwrapper.Launch.launch(Launch.java:135)
	at net.minecraft.launchwrapper.Launch.main(Launch.java:28)

**What happens / happened?**: Trying to access ''button 3'' for more info on Guardian Armor as it says see info. tip.   Something mentioned in issue: #101  similar to it but not quite.

**How can I reproduce it?**: unknown but suggest trying playing 1.10.2 with armorplus, dragonlib & jei?

**(Optional) Mod List**: Just started up jei_1.10.2-3.14.5.406

**Comment by sokratis12GR** [comment]
@Iteries could you link me your config file ? (for armorplus) armorplus,cfg

**Comment by sokratis12GR** [comment]
I can't seem to reproduce it but I'll release a fix for it to not happen if it is happening

**Comment by Iteries** [comment]
Can close this or a bit of help on adding another potion effect to guardian armor.  I went through the config again and realized that adding night_vision with power 0  was messing with it.   Perhaps is how I added it by doing this:  
            S:guardianArmorAddPotionEffect=water_breathing,night_vision
            I:guardianArmorEffectLevel=0,0
I even tried rearranging it by example of the ultarmor as this:
            S:guardianArmorAddPotionEffect <
                water_breathing
                night_vision>
             I:guardianArmorEffectLevel <
                0
                0>

Then did tried again by going through mod config system but only accepts 1 power.  Just felt that guardian armor should have night_vision as well.  I did read that we can modify armor to personal taste.  I know armor base value for toughness and 'plating' but didn't realize the set effects can't add more too, except maybe the ultarmor.   (personal note, I never used JEI before and didn't want too but apparently have too.  Changed a lot of armor recipes over the stretched MC versions, sorry for bothering)
